report.Rmd

```diff
@@ -642,7 +643,24 @@ serp_iw %>%
     title = "Number of SERPs' sister-search sidebar results by source, position, test group and wiki"
   ) +
   facet_grid(wiki ~ position, scale = "free_y") +
-  theme_facet(axis.text.x = element_text(angle = 90))
+  theme_min(axis.text.x = element_text(angle = 90))
```

Comment:
`theme_facet` adds the boxes around the facets that makes it easier to distinguish the wiki-position pairs.